app/decorators/order_decorator.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class OrderDecorator < ApplicationDecorator
+  delegate :name, :email, to: :user, prefix: true
+
+  def self.collection_decorator_class
+    OrdersAmountDecorator
+  end
+
+  def creation_date
+    created_at.strftime('%Y-%m-%d %H:%M:%S').to_s
```

Comment:
you could use `I18n.l` function for this. This way you'd get a general solution for the whole application in the one place.
https://www.rubydoc.info/github/svenfuchs/i18n/master/I18n/Backend/Base:localize